### ISSUE TITLE
Add legislationProposedOnly property on Legislation

### ISSUE THREAD

**Issue by debsidian** [schemaorg/suggestions-questions-brainstorming#65]
To indicate that the content is a only proposed legislation and not passed legislation.

Boolean key/value pair:
"legislationProposedOnly": "true"
"legislationProposedOnly": "false"

This way legislators and candidates can put forward policy proposals without any need to create a new schema.org subtype.

**Comment by RichardWallis** [comment]
See issue #7 for the context of the move from the main Schema.org [issue tracker](https://github.com/schemaorg/schemaorg/issues) to this repository.

**Comment by tfrancart** [comment]
I agree on the need to capture/describe draft legislation; but maybe not on the proposed solution. As a reference : ELI-DL is an ontology to describe "draft" / "proposed" legislation, [just published in a first stable version](https://joinup.ec.europa.eu/collection/eli-european-legislation-identifier/solution/eli-ontology-draft-legislation-eli-dl/release/final1).
In ELI-DL we have chosen to create a completely separate type to clearly distinguish between the enacted legal corpus and the not (yet) enacted texts, otherwise there is a risk for data consumers to mix both. Not sure we need to do the same in schema.org. If a simple property is added, I would rather rely/extend "creativeWorkStatus" rather than introducing another boolean.

This can open up a can of worms because the next step would be to describe the legislative projects themselves (like [a random EU procedure page](https://eur-lex.europa.eu/legal-content/EN/HIS/?uri=CELEX:52019PC0581&qid=1595520570324))

Also see current proposed enhancement of Legislation description at #24
